olp-cpp-sdk-authentication/include/olp/authentication/TokenProvider.h

```diff
@@ -80,8 +96,12 @@ class OLP_SDK_DEPRECATED("Deprecated. Will be removed in 04.2020")
   }
 
   /**
-   * @brief Allows the http status code of the last request to be retrieved.
-   * @returns http status code
+   * @brief Gets the HTTP status code of the last request.
+   * Return the HTTP code of the last token request if it was successful.
+   * Otherwise, returns the HTTP 503 Service Unavailable server error.
```

Comment:
What's the point of duplicating the same info in the @returns statement? Is is required by HERE docs?

Reply:
Deleted from the `brief`